backend/src/controller/listing.ts

```diff
@@ -3,6 +3,34 @@ import ExampleListingList from "../constants/ExampleListingList";
 import ExampleUserList from "../constants/ExampleUserList";
 import productsArray from "../controller/product";
 
+<<<<<<< HEAD
+const listingList = ExampleListingList;
+
+let fullInfoListings = new Array();
+function fillFullInfo() {
+  fullInfoListings = new Array();
```

Comment:
This function could be improved. First, you're setting one variable to an empty array twice.
Second, since `.map()` returns a new array, you don't have to mutate a var that's out of scope.
```
const fillFullInfo = () => listingList.map((listing) => {
  // code with .find call
  return { // object that you're currently pushing}
}
```

and then you can do `const fullInfoListings = fillFullInfo()` or you don't even have to assign it to a variable, but use this function in the endpoint :)